src/main/java/com/hillel/items_exchange/dto/AdvertisementDto.java

```diff
@@ -15,26 +15,26 @@
 @AllArgsConstructor
 @EqualsAndHashCode
 public class AdvertisementDto {
-    @PositiveOrZero(message = "{invalid.id}")
+    @PositiveOrZero(message = "Id value has to be 0 or positive")
```

Comment:
You should make correct merge of master branch here!

Reply:
My bad. Resolved